public/data/shubhambaghel.json

```
@@ -0,0 +1,23 @@
+{
+    "name": "Shubham Baghel",
+    "type": "personal",
+    "bio": "Full Stack developer and Open Source Enthusiast!",
+    "avatar": "https://avatars.githubusercontent.com/u/21123907?v=4",
```

Let's use this link so that the image updates automatically. Plus, the link is much shorter.
```suggestion
    "avatar": "https://github.com/shubhambaghel34.png",
```